azcredentials/internal/maputil/maputil.go

```diff
@@ -0,0 +1,87 @@
+package maputil
+
+import "fmt"
+
+func GetMap(obj map[string]interface{}, key string) (map[string]interface{}, error) {
```

Comment:
shall we add test for the utility functions?

Reply:
Added tests.